docs/editors-guide/merging-and-obsoleting.md

```diff
@@ -46,34 +46,43 @@ See [GitHub Discussion](https://github.com/monarch-initiative/mondo/discussions/
     1. Mondo ID for the term that will replace the obsoleted terms
     1. Label for the term that will replace the obsoleted terms
     
-### Workflow video explanation
-See [here](https://drive.google.com/file/d/1pnf0BT1qJ9wxXKvykkWhjwvFLceI2U-2/view?usp=sharing)
-    
-#### If new obsoletion reasons are needed
+    #### Video explanation of Workflow
+
+    See video explanation <a href="https://drive.google.com/file/d/1pnf0BT1qJ9wxXKvykkWhjwvFLceI2U-2/view?usp=sharing" target="_blank">here.</a>
+
+  
+    #### Request new obsoletion reasons as needed
 
-Request from the technical team if you need additional obsolescence reasons. This involves:
-   
-1. amending the QC check 
-2. updating the imports/removeseed.txt file and 
-3. the omo_import.owl goal. 
+    Request new obsoletion reasons from the technical team:
+    1. amending the QC check 
+    2. updating the imports/removeseed.txt file and 
+    3. the omo_import.owl goal. 
 
-Unfortunately, this complex process is necessary due to limitations with the OBO format, see [https://github.com/owlcollab/oboformat/issues/139](https://github.com/owlcollab/oboformat/issues/139).
+    Unfortunately, this complex process is necessary due to limitations with the OBO format, see [https://github.com/owlcollab/oboformat/issues/139](https://github.com/owlcollab/oboformat/issues/139).
 
-#### Monthly (before release)
+
+### Monthly (before Mondo release)
 1. Run the pipeline to [merge the ROBOT template](https://mondo.readthedocs.io/en/latest/editors-guide/robot-template/) and commit and merge the PR into mondo-edit.obo.
-3. Run the [risky_obsoletion_check](https://mondo.readthedocs.io/en/latest/editors-guide/risky_obsoletion_check) pipeline to check if external ontologies and databases (such as EFO and ClinGen) are using this Mondo term.
-  - if so, create a ticket on the [EFO tracker](https://github.com/EBISPOT/efo/issues) and notify ClinGen users (_for now, email Larry Babb_).
+1. Run the [risky_obsoletion_check](https://mondo.readthedocs.io/en/latest/editors-guide/risky_obsoletion_check) pipeline to check if external ontologies and databases (such as EFO and ClinGen) are using this Mondo term.
+    1. if any terms are reported, create a ticket on the [EFO tracker](https://github.com/EBISPOT/efo/issues) and notify ClinGen users (_for now, email Larry Babb_).
```

Comment:
do you want to include Larry's email here?
lbabb@broadinstitute.org

Reply:
nope, it's usually not a good idea to add someone's email in plain text on the internet since this can lead to lots of spam